scripts/genome_process.sh

```diff
@@ -57,37 +51,22 @@ rm -fr "$logFile"; touch "$logFile"
 ###  MAIN  ###
 ##############
 
-## GET & FILTER GENE ANNOTATIONS
-
-# Get genome sequences fasta files
-echo "Downloading genome sequences files..." >> "$logFile"
-# wget -i "${genomeSeqURLs}" --output-document "${rawDir}/${fileNamePrefix}.genome.fa.gz"
-# genomeSeq="${resDir}/${fileNamePrefix}.genome.fa.gz"
-
-for url in "${genomeSeqURLs[@]}"; do
-    wget "$url" --output-document "${rawDir}/$(basename "$url")" &> /dev/null
-done
-
-# Concatenate genome sequences fasta files
-echo "Concatenating genome sequences files..." >> "$logFile"
-genomeSeq="${resDir}/genome.fa.gz"
-for url in "${genomeSeqURLs[@]}"; do
-    cat "${rawDir}/$(basename "$url")" >> "$genomeSeq"
-done
-
-# Trim genome sequences IDs
+## TRIM GENOME SEQUENCES IDs
 echo "Triming genome sequenes IDs..." >> "$logFile"
+
 genomeSeqTrim="${resDir}/genome.processed.fa"
+cp "${genomeSeqFile}" "${output_dir}/genome.fa.gz"
+genomeSeq="${output_dir}/genome.fa.gz"
 gunzip "$genomeSeq"
-genomeSeq="${resDir}/genome.fa"
+genomeSeq="${output_dir}/genome.fa"
 awk '{if ($1 ~ /^>/) {print $1} else {print $0}}' "$genomeSeq" > "$genomeSeqTrim"
```

Comment:
So all this script actually does after removing the download and concatenation is this line. I don't think we actually need a script for that and can just create a Snakemake rule `trim_genome_seq_ids` with that `awk` command in an Ubuntu container (instead of rule `genome_process`).

You'd only need to change the name of the rule, remove the `script` input, and replace:

```console
"(bash {input.script} {params.dir_out} {log} {input.genome})"
```

with something like:

```console
"(zcat {input.genome} | awk '{if ($1 ~ /^>/) {print $1} else {print $0}}' - > {output.genome) &> {log}"
```

(untested)

And of course you can remove the script `genome_process.sh` altogether.